app/src/main/java/com/ciandt/dragonfly/example/data/ClassificationRepository.kt

```diff
@@ -0,0 +1,15 @@
+package com.ciandt.dragonfly.example.data
+
+import com.ciandt.dragonfly.example.data.mapper.ServicesToComparisonResult
+import com.ciandt.dragonfly.example.data.remote.RemoteDataSource
+import com.ciandt.dragonfly.example.features.feedback.model.ComparisonResult
+
+class ClassificationRepository(baseUrl: String) {
+
+    private val remoteDataSource = RemoteDataSource(baseUrl)
+
+    fun compareServices(image: String): ComparisonResult {
```

Comment:
I'd suggest to rename this method to "benchmark()". Think this is the term Simões is thinking of using in the UI.